src/collect/http/server.rs

```diff
@@ -265,7 +265,7 @@ impl RestHandler {
                     json.push(
                         jsonway::object(|json| {
                             json.set("reachable", false);
-                            json.set("url", "http://c/status".to_string());
+                            json.set("url", "http://foobar/status".to_string());
```

Comment:
Note to self: this exercises two cases in the visualizer.

1. Do not draw an edge from A to B if B is not a target of A.
1. Do not draw an edge from A to its target if the target is not a resource.